content/applications/finance/accounting/payables/supplier_bills/ocr.rst

```diff
@@ -1,56 +1,54 @@
-==============================================================
-Digitize vendor bills with optical character recognition (OCR)
-==============================================================
+=============================================================
+Digitalize documents with optical character recognition (OCR)
+=============================================================
 
-Encoding bills manually can be a time-consuming task. Having a solution that allows you to digitize
-them and automatically import the data into your database reduces errors and saves you time.
+Encoding vendor bills or customer invoices manually is a time-consuming task. With optical character
+recognition (OCR), digitalize and import these documents automatically into your database.
 
-.. important::
-   `Try it out <https://www.odoo.com/app/invoice-automation>`_! Upload one of your Bills or try one of our samples.
-   
-Set up the feature
-==================
-
-Go to :menuselection:`Accounting --> Settings --> Bill Digitalization`, and choose whether the bills
-should be processed automatically (with OCR) or manually (on demand).
+.. seealso::
+   - `Try it out <https://www.odoo.com/app/invoice-automation>`_! Upload one of your bills or try
+     one of our samples to see how it works.
+   - :doc:`/applications/general/in_app_purchase`
 
-.. image:: ocr/setup_ocr.png
-   :align: center
-   :alt: Activate the feature going to the settings application in Odoo Accounting
+Configuration
+=============
 
-Start digitizing your bills
----------------------------
+Digitalization is an :doc:`In-App Purchase (IAP) </applications/general/in_app_purchase>` service
+that requires prepaid credits to work. To buy credits, go to :menuselection:`Accounting -->
+Configuration --> Settings --> Digitization` and click on :guilabel:`Buy credits`. Or go to
+:menuselection:`General Settings --> Contacts` and under :guilabel:`Odoo IAP` click on
+:guilabel:`View My Services`.
 
-Scan your bills and then go to :menuselection:`Accounting --> Vendors --> Bills` and upload your
-document. Based on your configuration, it either processes the documents automatically, or you
-need to click on *Send for Digitalization* to do it manually.
+.. note::
+   - If you are on Odoo Online (SAAS) and have the Enterprise version, you benefit from free trial
+     credits to test the feature.
 
-.. tip::
-   You can also create a vendor bill through the *Documents* app or by using an email alias on your
-   journals.
+In :menuselection:`Accounting --> Configuration --> Settings --> Digitization`, choose whether the
+:guilabel:`Vendor Bills` and :guilabel:`Customer Invoices` should be processed automatically (with
+OCR) or manually (on demand).
```

Comment:
- Again, I don't find the category "digitalization". You could mention the feature name (with guilabel ;) )
- I can't find the Customer Invoices one? Only the Vendor Bills one.
- automatically, or manually, .... or not at all ^^ 
![image](https://user-images.githubusercontent.com/46810843/174088442-c5e9ece4-4d4d-42fa-8752-f1723081db1c.png)
- why the "with OCR" between brackets? That's the character recognition system but I don't understand why you wrote it here.
- Since you're not listing the options as they're written, I guess you could maybe add a screenshot.

Reply:
updated